levelkdev/swapr-dapp
# Change Opyn URL.

# Summary

Fixes part of #829

Opyn list has been moved and is outdated/irrelevant. In this PR we are removing it.

# To Test

1. Start Swapr locally
2. Check the dev console

404 errors regarding Opyn should no longer be there.

I also didn't find a good list to replace. If there is such a thing from an institutional/non-individual source, I'm open to make the change.
